src/core/a-entity.js

```diff
@@ -833,8 +774,7 @@ var proto = Object.create(ANode.prototype, {
       var stateIndex = this.states.indexOf(state);
       if (stateIndex === -1) { return; }
       this.states.splice(stateIndex, 1);
-      this.mapStateMixins(state, bind(this.unregisterMixin, this));
-      this.emit('stateremoved', {state: state});
+      this.emit('stateremoved', state);
```

Comment:
Intentional? I assumed we want to discuss recommendations for an alternative state API outside of this PR, so breaking changes to the current API don't seem immediately useful.

Reply:
We can save an object in case the API does stick around for a version or further even if an application is not using it. It's not looking like there'll be consensus resulting in removing this API at the moment.